### PR TITLE
Add Ghost.app 0.3.2

### PR DESCRIPTION
**Pull request by optikfluffel**
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download ghost-desktop` is error-free.
- [x] `brew cask style --fix ghost-desktop` left no offenses.
- [x] `brew cask install ghost-desktop` worked successfully.
- [x] `brew cask uninstall ghost-desktop` worked successfully.

### CODE REVIEW

**Comment by optikfluffel** [comment]
I don't really know what happened to [Job #40432.1](https://travis-ci.org/caskroom/homebrew-cask/jobs/126226878), but it failed after `store build cache`.
[Job #40432.2](https://travis-ci.org/caskroom/homebrew-cask/jobs/126226881) worked fine.

**Comment by adidalal** [comment]
Travis fails sometime, it happens. Will restart as needed.

Also, mind adding an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.mdt?

**Comment by optikfluffel** [comment]
@adidalal ✅ (with [same error at Travis](https://travis-ci.org/caskroom/homebrew-cask/jobs/126312903#L605) xD)

**Comment by vitorgalvao** [comment]
Looks like Travis is moody today, failing one of the tests on all PRs, despite everything running fine.

File: Casks/ghost-desktop.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'ghost-desktop' do
```
Comment (vitorgalvao) [inline Casks/ghost-desktop.rb]: Did not follow the token [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md). This needs to be `cask 'ghost' do`.

File: Casks/ghost-desktop.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  version '0.3.2'
+  sha256 '5c167e665a208847ffb5e01bb6e8927c6569bca00997b0d50314ef703cb89afe'
+
+  url "https://github.com/TryGhost/Ghost-Desktop/releases/download/v#{version}/ghost-desktop-#{version}-osx.zip"
```
Comment (vitorgalvao) [inline Casks/ghost-desktop.rb]: Needs a `# github.com/TryGhost/Ghost-Desktop was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in f293acc1d766c920fd604d81498545c7c8705d2c. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**